### DISCUSSION TITLE
Video Guide

### DISCUSSION THREAD

**Discussion by orassayag** [nicymike/serflings#136] [Ideas]
First of all, You are doing an awesome job, I'm wathcing the changelog each day and follow your update.

Serfcity was one of my favorites games when I was a child and I would like to go back and play it once again on my Lenovo. 

A video on Youtube showing step by step from skratch (Where to download the original game from in order to play with your serfling version of the game) will be much appreciated!

Thanks!

**Comment by nicymike** [comment]
Thank you. :)

Someone made a video about how to setup Serflings: https://www.youtube.com/watch?v=X6nCIfZRmNc
He mentions the History Edition and so will I. There may be sites out there where you can download the original game for free but I prefer to pay if the author is still selling it.
There are also german tutorials on youtube.

I will convert this issue to discussion because it is not really an issue on the game itself.
